dts/arm/st/f4/stm32f410.dtsi

```diff
@@ -91,6 +96,13 @@
 			#io-channel-cells = <1>;
 		};
 
+		rng: rng@50060800 {
+			compatible = "st,stm32-rng";
+			reg = <0x50060800 0x400>;
+			interrupts = <80 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x00000040>;
```

Comment:
Bit 31 RNGEN: RNG clock enable of the RCC AHB1 peripheral clock enable register : RCC_AHB1ENR